packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -54,7 +54,7 @@ const Info = observer(({ handleSelect, item, selected_value, list, info_banner }
             i.value !== RISE_FALL_EQUAL && i.value !== TURBOS.SHORT && i.value !== VANILLA.PUT
     );
     const has_toggle_buttons = /accumulator|turbos|vanilla|multiplier/i.test(selected_contract_type);
-    const should_show_video = /accumulator|turbos|vanilla|high_low|rise_fall|touch/i.test(selected_contract_type);
+    const should_show_video = !/match_diff|even_odd|over_under/i.test(selected_contract_type);
```

Comment:
I think it's better to keep the previous version because if we add a new contract type we may forget to add it here.

Reply:
Okay, will revert)